.circleci/config.yml

```diff
@@ -41,57 +41,57 @@ jobs:
           if [ "${CIRCLE_BRANCH}" == "master" ]; then
             ./architect deploy
           fi
-
-
-
-  e2eTestCurPRDraining:
+  
+  
+  
```

Comment:
The diff here is a mess because of ordering the tests alphabetically. But apart from the renaming nothing changed.